### PR TITLE
[FIO internal] arch: arm: mach-imx: fiohab: extend hab event message

### PR DESCRIPTION
**Pull request by vanmaegima**
In case of any hab events found in the image, print a message to help
with the most common errors (SPL incorrectly signed or board not
fused).

Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>

Please do not submit a Pull Request via github.  Our project makes use of
mailing lists for patch submission and review.  For more details please
see https://www.denx.de/wiki/U-Boot/Patches

### CODE REVIEW

**Review by MrCry0 — APPROVED** [review]
LGTM

File: arch/arm/mach-imx/fiohab.c [hunk 1]
```diff
@@ -192,6 +192,8 @@ static int hab_status(void)
 		       HAB_SUCCESS) {
 			if (!is_known_rng_fail_event(event_data, bytes)) {
 				printf("HAB events active error\n");
+				printf("Make sure the SPL is correctly signed \
+					and the board is fused\n");
```
Comment (ricardosalveti) [inline arch/arm/mach-imx/fiohab.c]: I think the end result on console will also contain this large empty space between the lines. You can concatenate by having "...signed " (new line) "and the board is fused\n".

Comment (MrCry0) [reply arch/arm/mach-imx/fiohab.c]: The kernel practice recommends not to split long strings at all. This prevents breaking a full-text searching thru code.

File: arch/arm/mach-imx/fiohab.c [hunk 2]
```diff
@@ -200,6 +202,8 @@ static int hab_status(void)
 		return 0;
 #else
 		printf("HAB events active error\n");
+		printf("Make sure the SPL is correctly signed \
```
Comment (MrCry0) [inline arch/arm/mach-imx/fiohab.c:205]:
```suggestion
		printf("Make sure the SPL is correctly signed and the board is fused\n");
```

Comment (MrCry0) [inline arch/arm/mach-imx/fiohab.c]:
```suggestion
		printf("Make sure the SPL is correctly signed and the board is fused\n");
```

File: arch/arm/mach-imx/fiohab.c [hunk 3]
```diff
@@ -200,6 +202,8 @@ static int hab_status(void)
 		return 0;
 #else
 		printf("HAB events active error\n");
+		printf("Make sure the SPL is correctly signed and the board is fused\n");
+			and the board is fused\n");
```
Comment (MrCry0) [inline arch/arm/mach-imx/fiohab.c]:
```suggestion
```

**Review by MrCry0 — APPROVED** [review]
LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Pull request merged**